x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_list_page/components/agent_status_filter.tsx

```diff
@@ -0,0 +1,202 @@
+/*
```

Comment:
I have moved the agent status filter to its own component as it has got a bit more complex

Reply:
Thanks for doing it!